sql-ballerina/tests/utils.bal

```diff
@@ -17,26 +17,38 @@
 import ballerina/io;
 import ballerina/file;
 import ballerina/lang.runtime as runtime;
-import ballerina/os;
```

Comment:
If this dependency is not used, lets remove it from the build

Reply:
another module may be using it. can not remove at the moment.